README.md

```diff
@@ -149,18 +149,21 @@ In addition to this, you will need to grant access to the CircleCI computers
 to push to your documentation repository. To do this, visit the Project Settings
 page of your project in CircleCI. Select `Checkout SSH keys` option and then
 choose `Create and add user key` option. This should grant CircleCI privileges
-to push to the repository `https://github.com/USERNAME/DOC_REPO/`.
+to push to the repository `https://github.com/USERNAME/DOC_REPO/`
+(see: https://circleci.com/docs/adding-read-write-deployment-key/; make sure
```

Comment:
The formatting could be a little better. Also to be consistent with the rest of the documentation, it would be better if we had a named link here, for eg: 
`[Circle CI documentation on ssh keys](...url...)`